aws/policy/security-services.yaml

```diff
@@ -116,15 +139,15 @@ Statement:
       - iam:TagRole
       - iam:UntagRole
     Resource:
-      - arn:aws:iam::{{ aws_account_id }}:role/*
+      - 'arn:aws:iam::{{ aws_account_id }}:role/*'
   - Sid: AllowPassRole
     Effect: Allow
     Action:
       - iam:PassRole
     Resource:
-      - arn:aws:iam::{{ aws_account_id }}:role/ansible_lambda_role
-      - arn:aws:iam::{{ aws_account_id }}:role/ansible-test-sts-*
-      - arn:aws:iam::{{ aws_account_id }}:role/ansible-test-*
+      - 'arn:aws:iam::{{ aws_account_id }}:role/ansible-test-*'
+      # Legacy - We need to backport ansible-collections/community.aws/63
```

Comment:
I'm less sure that one will be accepted as a backport.  I'm not opposed to you proposing it; I'd +1 it; but it will be ultimately up to the RM.

Reply:
It's a CI only change, depending on when we stop testing against Ansible 2.9 it's probably not worth the effort.  Updated the comment